doc/api/crypto.md

```diff
@@ -1,5 +1,7 @@
 # Crypto
 
+<!--introduced_in=v0.10.0-->
```

Comment:
Oh cool, I had only been looking at https://nodejs.org/dist/ and must have messed up a few versions. I'll go back over all the docs and make sure these tags are the right version number.